cypress.json

```diff
@@ -1,4 +1,5 @@
 {
+  "projectId": "odida5",
```

Comment:
Can we customize this in cypress?

Reply:
I don't think so. This project ID is generated by the Cypress Dashboard when adding a project to an organization. The local Cypress runner can be connected to your Cypress account and this project ID is used to link a project to the Dashboard. From there a record key is generated (projectID is public, the record key is private)

Reply:
Okay no worries, was just wondering for URL purposes.